### PR TITLE
Add-on GUID needs to be 4chars or more (issue #1007)

### LINKED ISSUE #1007: Add-on GUID must be 4chars or more
Trunion (the signing server) needs the add-on ID to be [4 chars or more](https://github.com/mozilla/trunion/blob/master/trunion/validators.py#L167-L170):

`uwsgi-prelim-amo-signer: HTTPBadRequest: addon_id is very short(<4 bytes): "@pg"`

### PR DESCRIPTION
**Pull request by magopian**
Fixes #1007

### CODE REVIEW

File: apps/devhub/tests/test_views_validation.py [hunk 1]
```diff
@@ -107,6 +107,19 @@ def test_long_uuid(self):
         xpi_info = check_xpi_info({'guid': long_guid, 'version': '1.0'})
         assert xpi_info['guid'] == long_guid
 
+    def test_too_short_uuid(self):
+        """An add-on uuid needs to be 4 chars or more, see issue #1007."""
+        with self.assertRaises(forms.ValidationError) as exc:
+            check_xpi_info({
+                'guid': u'@fo'})
```
Comment (EnTeQuAk) [inline apps/devhub/tests/test_views_validation.py]: That indentation looks very wrong

**Comment by EnTeQuAk** [comment]
r+wc

File: apps/devhub/tests/test_views_validation.py (hunk 1 above)
Comment (magopian) [inline apps/devhub/tests/test_views_validation.py]: thanks, fixed

**Comment by magopian** [comment]
The xpi file change is because the manifest had a 3chars long guid.

**Comment by jvillalobos** [comment]
This is inconsistent with what Firefox accepts, so it will cause developers problems after they've tested their add-on in Firefox and want to submit it. Also, there are existing add-ons with short IDs that would be broken because of this.

This change needs to be discussed at the Product level before it can be accepted.

**Pull request closed without merging**